bin/main.rb

```diff
@@ -1,2 +1,74 @@
 #!/usr/bin/env ruby
-puts 'Hello world!'
+
+puts 'INSTRUCTIONS'
+puts '---------------------------------------------------'
+puts 'A number represents each square spaces on the board'
+puts 'from 1 up to 9. To make a move specifies a number'
+puts 'in the range of possible numbers at each step into'
+puts 'the game. It\'s that simple :). Enjoy!!!'
+
+puts 'Introduce the name of the first player.'
+player_one = gets.chomp
+
+puts 'Introduce the name of the second player'
+player_two = gets.chomp
+start = 1
+
+# now i'll display the board and an array whith each available moves
+
+while start.positive?
```

Comment:
- The while should check a variable that defines a winner or have a maximum move for a draw, for example. You can fake the behavior of the variable while you don't have the logic, but it needs to be ready to receive the logic.